### PR TITLE
Organization Rename

### PR DESCRIPTION
**Pull request by endzyme**
Fixing up the readme and some references to badges on the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ pip install reckoner
 
 Via Git
 ```shell
-pip install git+https://github.com/reactiveops/reckoner@master
+pip install git+https://github.com/FairwindsOps/reckoner@master
```
Comment (sudermanjr) [inline README.md]: In Go I think we have been using lowercase, but @bobby-brennan should verify.

Comment (bobby-brennan) [reply README.md]: Per offline discussion, we're gonna go with lowercase for golang packages, but for copy/paste stuff from READMEs, let's keep it `FairwindsOps`

File: README.md [hunk 2]
```diff
@@ -1,21 +1,21 @@
...
+# reckoner [![CircleCI](https://circleci.com/gh/FairwindsOps/reckoner.svg?style=svg)](https://circleci.com/gh/FairwindsOps/reckoner) [![codecov](https://codecov.io/gh/FairwindsOps/reckoner/branch/master/graph/badge.svg)](https://codecov.io/gh/FairwindsOps/reckoner)
 
 Command line helper for helm.
 This utility adds to the functionality of [Helm](https://github.com/kubernetes/helm) in multiple ways:
 * Creates a declarative syntax to manage multiple releases in one place
 * Allows installation of charts from a git commit/branch/release
 
-**Want to learn more?** ReactiveOps holds [office hours on Zoom](https://zoom.us/j/242508205) the first Friday of every month, at 12pm Eastern. You can also reach out via email at `opensource@reactiveops.com`
+**Want to learn more?** ReactiveOps holds [office hours on Zoom](https://zoom.us/j/242508205) the first Friday of every month, at 12pm Eastern. You can also reach out via email at `opensource@fairwinds.com`
```
Comment (bambash) [inline README.md]: Fairwinds holds**

**Comment by bobby-brennan** [comment]
I think this is obviated by #127

**Pull request closed without merging**